frame/evm-system/src/tests.rs

```diff
@@ -8,9 +8,9 @@ use sp_core::H160;
 
 use crate::{mock::*, *};
 
-/// This test verifies that creating account works in the happy path.
+/// This test verifies that creating evm managed account works in the happy path.
```

Comment:
```suggestion
/// This test verifies that creating evm-managed account works in the happy path.
```
Should be like in similar places too

Reply:
Done